portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java

```diff
@@ -727,7 +763,7 @@ protected long getLastModified(HttpServletRequest httpServletRequest) {
 				FileEntry fileEntry = null;
 
 				try {
-					fileEntry = getFileEntry(pathArray);
+					fileEntry = getFileEntry(pathArray, httpServletRequest);
```

Comment:
You're passing the whole request, but you only need the permission checker. Pass the the permission checker directly.

Also, as you're changing the method signature, make the method private (it will affect semver in the same way).

Reply:
the method will change, so I need the request
but I will refactor to extract a method.

Thanks!